inspectit-oce-agent/src/system-test/java/rocks/inspectit/oce/TestUtils.java

```diff
@@ -0,0 +1,32 @@
+package rocks.inspectit.oce;
+
+import io.opencensus.impl.internal.DisruptorEventQueue;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class TestUtils {
+
+    /**
+     * Opencensus internally manages a queue of events.
+     * We simply add a event to the queue and wait unti it is processed.
+     */
+    public static void waitForOpenCensusQueueToBeProcessed() {
+        AtomicBoolean entryProcessed = new AtomicBoolean(false);
```

Comment:
why here both Atomic and sync? simply have atomic only and instead of wait/notify do a small sleep until you get true from the atomic?

Reply:
Atomic is just used to have a settable boolean.